rpc/wrtc_signaling_answerer.go

```diff
@@ -160,32 +148,15 @@ func (ans *webrtcSignalingAnswerer) startAnswerer() {
 		md := metadata.New(nil)
 		md.Append(RPCHostMetadataField, ans.hosts...)
 		md.Append(HeartbeatsAllowedMetadataField, "true")
-		answerCtx := metadata.NewOutgoingContext(ans.closeCtx, md)
+		answerCtx := metadata.NewOutgoingContext(ans.bgWorkers.Context(), md)
```

Comment:
[nit] Can we add a small comment above this clarifying why we're using `bgWorkers.Context()`?